src/mocha.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { telemetry } from './telemetry';
```

Comment:
I suggest that we reconsider the name of this file. `.spec.ts` indicates a test suite, but we're not testing anything here technically. What if we moved it to `utils/tests.ts`?

Reply:
Great idea @waldekmastykarz 👍 will do so